src/View/Admin/adminMainPage.html.twig

```diff
@@ -4,17 +4,19 @@
 
 {% block content %}
     <h1>Administration du site</h1>
-    <p>
-        <a href="index.php?route=admincolor">Administrer les couleurs</a>
-    </p>
+    <ul>
+        <li><a href="index.php?route=admincolor">Administrer les couleurs</a></li>
+        <li><a href="index.php?route=adminmodels">Administrer les modèles</a></li>
+    </ul>
 
 
     {# TODO A faire :
```

Comment:
le todo aurait plus sa place dans le controlleur !

Reply:
move to adminController